macros/MatrixReduce.pl

```diff
@@ -14,89 +14,105 @@ =head1 SYNOPSIS
 
 =over 12
 
-=item Get the reduced row echelon form: C<$Areduced = rref($A);>  Should be used in the fraction context with all entries of $A made into fractions.
+=item Get the reduced row echelon form: C<$Areduced = rref($A);>
+  
+Should be used in the fraction context with all entries of $A made into fractions.
 
-=item Make matrix entries do fraction arithmetic (rather than decimal arithmetic): After selecting the Fraction context using Context('Fraction')->parens->set("[" => {formMatrix => 1}), C<$A = apply_fraction_to_matrix_entries($A);> applies Fraction() to all of the entries of $A, which makes subsequent matrix algebra computations with $A use fraction arithmetic.
+=item Make matrix entries do fraction arithmetic (rather than decimal arithmetic): 
+
+After selecting the Fraction context using Context('Fraction')->parens->set("[" => {formMatrix => 1}), C<$A = apply_fraction_to_matrix_entries($A);> applies Fraction() to all of the entries of $A, which makes subsequent matrix algebra computations with $A use fraction arithmetic.
 
 =item Get the reduced column echelon form: C<$Areduced = rcef($A);>
 
-=item Change the value of a matrix entry: C<change_matrix_entry($A,[2,3],50);> changes the [2,3] entry to the value 50.
+=item Change the value of a matrix entry: C<change_matrix_entry($A,[2,3],50);> 
+
+changes the [2,3] entry to the value 50.
 
 =item Construct an n x n identity matrix: C<$E = identity_matrix(5);>
```

Comment:
Note:  there is already a MatahObject method for that:  `Value::Matrix->I(5)`.

Reply:
True, but I believe it was added after MatrixReduce.pl was created. I've added a deprecated statement to this for now.  Eventually I'd like to consolidate functions in MatrixReduce, and in Tableau and in Peter @Selinger 's linear algebra macros. Some of them should go into Value::Matrix -- the others should be gathered into a small number of macro files.